Quentinchampenois/decidim-module-budgets_importer
# bump!: Release 2.0.0 - Decidim dependency to v0.27

#### Description

Bump module to work with Decidim 0.27

#### Note
🚨 **Breaking change**
Between version `0.26` and `0.27` file upload has changed to Direct Upload. Module now supports direct upload

#### How to test 

Install dev app :
* `bundle install`
* `bundle exec rake development_app`
* `cd development_app && bundle exec rails s`


* Log in as admin
* Go to BO
* Navigates to budgets in a participatory process
* In budget navigates to related projects
* In projects index, Export all projects
* Then select `Import projects to budget` in the `import` dropdown
* Upload the previously exported file
It should work and you should be redirected to the projects index

**Try to generated errors in file, you can remove required params, link to bad ids etc**

#### Tasks

- [x] Use OpenSourcePolitics Github Actions in CI
- [x] Add Release and Publish Actions
